cves/kernel/CVE-2020-11494.yml

```diff
@@ -326,8 +327,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: The fix commit was signed of by Richard Palethorpe <rpalethorpe@suse.com>, David S. Miller <davem@davemloft.net>, Ben Hutchings <ben@decadent.org.uk>
```

Comment:
- "commit was signed of by" of should be off
- include "and" after David Miller in the developer list

Reply:
I'd also add the link to the discussion here.